src/components/Elements/Card.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
I have extracted this code into its own file

Reply:
I missed that part